GenerateGridJson/grid.json

```diff
@@ -1,7 +1,5 @@
 [
```

Comment:
should this file even be part of the PR?

Reply:
It makes it easier for a person who is trying to build and run locally. I could move it to a "samples" folder or something like that.